### ISSUE TITLE
[BUG] PostgreSQL cannot see triggers, sequences and types

### ISSUE THREAD

**Issue by diogomrts** [cweijan/vscode-mysql#1265]
In older versions of PostgreSQL(13.13) we were not allowed to use aliases without the AS keyword:

```SQL
SELECT name name_alias FROM users; -- This will fail

SELECT name  AS name_alias FROM users; -- This works
```

When trying to select TRIGGERS for example we get this error: 

```
2024-09-10 16:10:21 [ERROR] Execute fail : SELECT TRIGGER_NAME "trigger_name",event_object_table table_name,string_agg(event_manipulation, ',') event FROM information_schema.TRIGGERS WHERE 
    trigger_schema = 'public'  AND event_object_table='blacklist' GROUP BY event_object_table,trigger_name ORDER BY TRIGGER_NAME ASC
```

Adding the AS keyword to this query we get a working one:

```SQL
SELECT TRIGGER_NAME "trigger_name", event_object_table as table_name, string_agg(event_manipulation, ',') as event FROM information_schema.TRIGGERS WHERE 
    trigger_schema = 'dashboards'  AND event_object_table='public_widgets_presets' GROUP BY event_object_table,trigger_name ORDER BY TRIGGER_NAME ASC
```

PostgreSQL version: 13.13

**Comment by cweijan** [comment]
Thanks for the feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
Fixed in version 7.6.1.
